packages/ui/src/app/pages/Forum/ForumCategory.tsx

```diff
@@ -99,13 +101,15 @@ export const ForumCategory = () => {
         <>
           {!!category.subcategories.length && (
             <RowGapBlock gap={24}>
-              <ItemCount count={category.subcategories.length}>
-                {isArchive ? 'Archived categories' : 'Categories'}
-              </ItemCount>
-              <ForumCategoryList categories={category.subcategories} isArchive={isArchive} />
+              <HorizontalScroller
+                title={isArchive ? 'Archived categories' : 'Categories'}
+                items={category.subcategories.map((category) => (
+                  <CategoryCard key={category.id} category={category} />
+                ))}
+                count={category.subcategories.length}
```

Comment:
design shows 2

Reply:
I fixed this problem and updated it. please check it again.